src/otx/algorithms/visual_prompting/adapters/openvino/model_wrappers/openvino_adapters.py

```diff
@@ -0,0 +1,169 @@
+"""Openvino Adapter Wrappers of OTX Visual Prompting."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from functools import partial
+from typing import Tuple
+
+import numpy as np
+import openvino.runtime as ov
+from openvino.model_api.adapters import OpenvinoAdapter
+from openvino.preprocess import ColorFormat, PrePostProcessor
+from openvino.runtime import Output, Type
+from openvino.runtime import opset10 as opset
+from openvino.runtime.utils.decorators import custom_preprocess_function
+
+
+def resize_image_with_aspect_pad(input: Output, size, keep_aspect_ratio=True, interpolation="linear", pad_value=0):
```

Comment:
In MAPI resize with AR & pad is called fit_to_window mode. Why it can't be used?

Reply:
As I mentioned in summary, there is an implementation bug and it was applied in v0.1.3. (https://github.com/openvinotoolkit/model_api/pull/100).
But otx 1.4.0 uses v0.1.2 and this fix is not applied.
So, I temporarily brought those parts to use `fix_to_window`.
If model api version is updated, I'll remove them and use modules in model api.

Reply:
Since we had passed FF milestone, barely we can introduce such changes like deps upgrades. I'm even confused why do we merge this PR after FF.
@goodsong81 could you comment here?

Reply:
Visual prompting task was planned to be added in 1.4.0, but there are so many considerations and it has been taking time.
So, train/eval/export tasks were done before FF, and deploy/optimize (pot) will be finished before CF.

Reply:
Hi @sovrasov, I am sorry that you missed the development schedule for the visual prompting task. It was agreed to merge `deploy` and `optimize` later FF under the considerations of its limited impacts to Geti. We plan to deliver the visual prompting task by 1.4.0 because this is the most basic preparation for the demo of one-shot transfer SAM.

Reply:
Ok thanks for comments. Indeed, we have some issues with the communicating release-related decisions within the team.
I've created an issue for the future refactoring: https://github.com/openvinotoolkit/training_extensions/issues/2339